src/overlays/actors/ovl_En_Tg/z_en_tg.h

```diff
@@ -2,15 +2,36 @@
 #define Z_EN_TG_H
 
 #include "global.h"
+#include "objects/object_mu/object_mu.h"
 
 struct EnTg;
 
 typedef void (*EnTgActionFunc)(struct EnTg*, PlayState*);
 
+typedef struct EnTgHeartInfo {
+    /* 0x00 */ u8 isFirstHeartSpawned; // Whether or not the first heart in the animation is spawned
```

Comment:
```suggestion
    /* 0x00 */ u8 enabled; // Whether or not the first heart in the animation is spawned
```
This looks like the actors specifics effect, which happens to be hearts. I know we've started standardizing these, but I'm not 100%.

Reply:
Thanks! I looked at a few other examples of Effects structs and renamed things similarly.